documentation/modules/rn-2.5.adoc

```diff
@@ -74,16 +74,17 @@ In addition to standard password authentication, the following authentication me
 
 The validation service includes default validation rules for virtual machines from OpenStack. link:https://issues.redhat.com/browse/MTV-508[(MTV-508)]
 
-
 .VDDK is now optional for VMware vSphere providers
 
 The VMware vSphere source provider can now be created without specifying a VDDK `init` image. It is strongly recommended to create a VDDK `init` image to accelerate migrations.
 
 .Deployment on OKE enabled
 
-// adding the link to the "About OKE" docs as it is better than nothing
 In {project-short} 2.5.3, deployment on OpenShift Kubernetes Engine (OKE) has been enabled. For more information, see link:https://docs.openshift.com/container-platform/4.14/welcome/oke_about.html[About OpenShift Kubernetes Engine]. link:https://issues.redhat.com/browse/MTV-803[(MTV-803)]
 
+.Migration of VMs from {rhv-short} to {virt} with encrypted RBD now supported
+
+In {project-short} 2.5.4, migration of VMs that have encrypted RADOS Block Devices (RBD) from {rhv-full} ({rhv-short}) to {virt} is now supported. link:https://issues.redhat.com/browse/MTV-851[(MTV-851)]
```

Comment:
need to mention that the configuration we've added need to change in this case (since by default the overhead is 0)

Reply:
we had the table of configurations, we can point to the table and add the new configuration value

Reply:
@ahadas Not sure which configuration you mean -- looking at  https://issues.redhat.com/browse/MTV-851, are you saying we need to tell the users to set  controller_block_overhead==2Gi, or do you have some other configuration in mind? Also, is that enough information -- will a user know how to set that parameter? I did not see anything similar in the user guide. Ping me on Slack if that's easier.

Reply:
@RichardHoch yes, I referred to controller_block_overhead - for encrypted RBD, the user can set it to 1Gi even
about `will a suer know how to set that parameter?` - you described it in the section that @liranr23 referred to above, so users are supposed to know but you can explicitly point them to that here